Paper.tex

```diff
@@ -610,6 +627,38 @@ \subsubsection{Block Header Validity}
 \nonumber& & H_{\mathrm{l}} \geqslant 5000
 \end{eqnarray}
 
+\linkdest{H__f}The \textit{London} release introduced the block attribute \textit{base fee per gas} \hyperlink{block_baseFeePerGas_H__f}{$H_{\mathrm{f}}$} (see EIP-1559 by \cite{EIP-1559}).
+The base fee is the amount of wei burned per unit of gas consumed while executing transactions within the block.
+The value of the base fee is a function of the difference between the gas used by the parent block and the parent block's \textit{gas target}.
+
+The expected base fee per gas is defined as $F(H)$:
+
+\begin{equation}
+  F(H) \equiv \begin{cases} 
+    1000000000  & \text{if} \quad H_{\mathrm{i}} = F_{\mathrm{London}} \\
+    {P(H)_{\mathrm{H}}}_{\mathrm{f}}  & \text{if} \quad {P(H)_{\mathrm{H}}}_{\mathrm{g}} = \tau \\
+    {P(H)_{\mathrm{H}}}_{\mathrm{f}} - \nu & \text{if} \quad {P(H)_{\mathrm{H}}}_{\mathrm{g}} < \tau \\
+    {P(H)_{\mathrm{H}}}_{\mathrm{f}} + \nu & \text{if} \quad {P(H)_{\mathrm{H}}}_{\mathrm{g}} > \tau \\
+  \end{cases}
+  \end{equation}
+
+where:
+  \begin{align}
+    \tau &\equiv \frac{{P(H)_{\mathrm{H}}}_{\mathrm{l}}}{\rho} \\
+    \rho &\equiv 2 \\
+    \nu &\equiv \begin{cases}
+      \frac{{P(H)_{\mathrm{H}}}_{\mathrm{f}} \times ({\tau - P(H)_{\mathrm{H}}}_{\mathrm{g}})}{\tau \times \xi} &  \text{if} \quad {P(H)_{\mathrm{H}}}_{\mathrm{g}} < \tau \\
```

Comment:
Because of the way how it spec-ed in EIP-1559 with integer division, the more accurate version is
floor(floor(nominator/tau)/xi)

Reply:
Yea I think you're right. I think using `floor` for division is taken for granted in the paper, but I think you're right on the order of operations here.

Reply:
I may try to double check this using live values.

Reply:
Confirmed the logic using live values (https://dune.com/queries/3232934?category=canonical) and updated this section.